scripts/deploy.args.sh

```diff
@@ -25,7 +25,7 @@ esac
 
 # URL used by II-issuer in the id_alias-verifiable credentials (hard-coded in II)
 # Represents more an ID than a URL
-II_VC_URL="https://identity.ic0.app"
+II_VC_URL="https://identity.internetcomputer.org"
```

Comment:
Side note: not idea what that is, I just assume the third party that requires this domain are backwards compatible. Just saying since I reviewed this PR.